impl/src/test/java/io/kroxylicious/testing/kafka/KafkaClusterTest.java

```diff
@@ -671,4 +699,14 @@ private void createClientCertificate() throws GeneralSecurityException, IOExcept
         this.clientKeytoolCertificateGenerator.generateSelfSignedCertificateEntry("clientTest@kroxylicious.io", "client", "Dev", "Kroxylicious.ip", null, null, "US");
     }
 
+    public static boolean zookeeperAvailable() {
```

Comment:
nit: this duplicates the implementation in `AbstractExtensionTest` should `KafkaClusterTest` extend `AbstractExtensionTest`?

I guess `private static final boolean ZOOKEEPER_AVAILABLE = AbstractExtensionTest.zookeeperAvailable();` is the other option